person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age: nil, parent_permission: true, name: 'Unknown')
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age && @age >= 18

```

Comment:
- [ ] @uno36 Great work on your of_age method however I would suggest instead of using && operator and you are not comparing ages just simplify it to `@age >= 18`, Ruby automatically returns false in case our age is below 18